src/molecules/message/Message.stories.js

```diff
@@ -1,17 +1,15 @@
 import { storiesOf } from '@storybook/vue'
 import { action } from '@storybook/addon-actions'
 
+import generateVueInfoTable from '../../../utils/helpers/generate-vue-info-table.js'
+import selectorsConfig from './Message.selectors.json'
+
 import AMessage from './Message.vue'
 
 const info = `
-  ---
-  #### Use these modifiers with \`.message\` class.
-  - \`.message--success\` - Selector for applying styles for success message
-  - \`.message--error\` - Selector for applying styles for error message
-  ---
+  ${generateVueInfoTable(selectorsConfig, 'BEM modifiers')}
 `
 
-// @vue/component
 const defaultData = {
   components: { AMessage },
```

Comment:
We need to add props `classKnobs` and pass options from `Message.selectors.json` to make it visible in storybook app, now they are not available to choose :cry:

Reply:
done